TTS/TikTok.py

```diff
@@ -1,101 +1,113 @@
-import base64
 from utils import settings
-import random
 import requests
 from requests.adapters import HTTPAdapter, Retry
 
-# from profanity_filter import ProfanityFilter
-# pf = ProfanityFilter()
-# Code by @JasonLovesDoggo
-# https://twitter.com/scanlime/status/1512598559769702406
+from attr import attrs, attrib
 
-nonhuman = [  # DISNEY VOICES
-    "en_us_ghostface",  # Ghost Face
-    "en_us_chewbacca",  # Chewbacca
-    "en_us_c3po",  # C3PO
-    "en_us_stitch",  # Stitch
-    "en_us_stormtrooper",  # Stormtrooper
-    "en_us_rocket",  # Rocket
+from TTS.common import BaseApiTTS, get_random_voice
+
+# TTS examples: https://twitter.com/scanlime/status/1512598559769702406
+
+voices = dict()
+
+voices['nonhuman'] = [  # DISNEY VOICES
+    'en_us_ghostface',  # Ghost Face
+    'en_us_chewbacca',  # Chewbacca
+    'en_us_c3po',  # C3PO
+    'en_us_stitch',  # Stitch
+    'en_us_stormtrooper',  # Stormtrooper
+    'en_us_rocket',  # Rocket
     # ENGLISH VOICES
 ]
-human = [
-    "en_au_001",  # English AU - Female
-    "en_au_002",  # English AU - Male
-    "en_uk_001",  # English UK - Male 1
-    "en_uk_003",  # English UK - Male 2
-    "en_us_001",  # English US - Female (Int. 1)
-    "en_us_002",  # English US - Female (Int. 2)
-    "en_us_006",  # English US - Male 1
-    "en_us_007",  # English US - Male 2
-    "en_us_009",  # English US - Male 3
-    "en_us_010",
+voices['human'] = [
+    'en_au_001',  # English AU - Female
+    'en_au_002',  # English AU - Male
+    'en_uk_001',  # English UK - Male 1
```

Comment:
Why use single quotes, like ever? Black-formatter would never let something like this through lmao

Reply:
because I hate to press `SHIFT` every time I want to place a bracket)

Reply:
Great but you didn't have to do anything to leave it the same You had to specially format it to go to single quotes

Also this project isn't about you.

Reply:
@JasonLovesDoggo 
yup, I know. I just used something I think is right 

I actually don't think that PR is the place to talk about personal preferences so if you really want single brackets to be gone I can revert it everywhere I put them

Reply:
@CordlessCoder 
If you are talking about linter they are configurable
If you talking about pre-commit Black it'll do nothing, I used to have it installed and configured in my projects

Just why double quotes? Why press SHIFT every time? It's not a jS, there is no difference

Reply:
> @CordlessCoder If you are talking about linter they are configurable If you talking about pre-commit Black it'll do nothing, I used to have it installed and configured in my projects
> 
> Just why double quotes? Why press SHIFT every time? It's not a jS, there is no difference

Because code standards exist. Single quotes are unreadable and formatting with black is required for making PRs, and it'd replace single quotes with double quotes automatically. You don't get to s**t over years of consistent code standardization because you're too lazy to press a key

Reply:
@CordlessCoder 
yo dude chill
I'm just saying there is no difference and you trying to insult me or whatever with imaginary rule

and here is the real one
> [String Quotes](https://peps.python.org/pep-0008/#string-quotes)
In Python, single-quoted strings and double-quoted strings are the same. This PEP does not make a recommendation for this. Pick a rule and stick to it. When a string contains single or double quote characters, however, use the other one to avoid backslashes in the string. It improves readability.

if you want to stick to something, just say it, I'll revert quotations everywhere and we can write it down in contribution guideline to avoid this in the future

Reply:
> @CordlessCoder yo dude chill I'm just saying there is no difference and you trying to insult me or whatever with imaginary rule
> 
> and here is the real one
> 
> > [String Quotes](https://peps.python.org/pep-0008/#string-quotes)
> > In Python, single-quoted strings and double-quoted strings are the same. This PEP does not make a recommendation for this. Pick a rule and stick to it. When a string contains single or double quote characters, however, use the other one to avoid backslashes in the string. It improves readability.
> 
> if you want to stick to something, just say it, I'll revert quotations everywhere and we can write it down in contribution guideline to avoid this in the future

Sorry for forcing you to endure being subjected to this imaginary rule. You could have looked at the entire codebase so far and after seeing that we have only used double quotes just use them as well

Reply:
@CordlessCoder 
If it's being used everywhere it doesn't mean that you obligated to use it. I just used what I liked because there is no rule about that and I wasn't expecting arguing about that. I'll revert it soon.

Can you add something in contribution guideline about quotation? I really don't want anyone to argue about it in the future